recipe/meta.yaml

```diff
@@ -14,19 +14,19 @@ source:
      - mmarchive.patch  # [win]
 
 build:
-  number: 5
+  number: 6
   skip: True  # [win32]
 
 requirements:
   build:
     - toolchain  # [unix]
```

Comment:
```suggestion
```

There should be no need to depend on toolchain  here anymore.

This should also be split into `build` and `host` requirements. From the looks of it, `tk` should be a host requirement, while the rest of the build requirements should stay in `build`:

```yaml
requirements:
  build:
    - {{ compiler('c') }}
    ...
  host:
    - tk
  run:
    - tk
```